obtrusive.txt

```diff
@@ -328,6 +328,7 @@ origo.hu##div[class="accept-cookie-open"]
 ouest-france.fr###disclaimerCookies
 out-law.com###cookie
 parcelmotel.com###cookiemonster
+parisinfo.com#cookieChoiceBanner
```

Comment:
This should have the `##` between the end of the domain and the start of the CSS selector.